packages/python/plotly/requirements.txt

```diff
@@ -7,3 +7,4 @@
 
 ## retrying requests ##
 tenacity>=6.2.0
+anywidget>=0.9.0
```

Comment:
Lots of proposed new requirements coming into Plotly.py... How does requiring AnyWidget affect the bundle size? Does it need to be a mandatory dependency, or is it safe to error out and ask the user to install it manually? 

Should we add e.g. `pip install plotly[notebook]` so we can _recommend_ rather than enforce AnyWidget install?

See: https://github.com/plotly/plotly.py/pull/4644

Reply:
Oh yeah actually it isn't bundled with `plotly.py` - I added an [error message](https://github.com/plotly/plotly.py/blob/4ab2a0cc6b70a5bd7f6b0348f8a7f5aa3834318a/packages/python/plotly/plotly/missing_anywidget.py) in this PR that tells users to download `anywidget` if they try to create a `FigureWidget` without `anywidget` installed. This is identical to the behavior with `ipywidgets` currently, except replaces `ipywidgets` with `anywidget`. `ipywidgets` and `anywidget` aren't significantly different sizes.

Reply:
I'll do testing locally to get a feel for the error cases

Reply:
> How does requiring AnyWidget affect the bundle size?

anywidget is a pure python and primarily wraps ipywidgets. 

```sh
anywidget v0.9.13
├── ipywidgets v8.1.5
│   ├── comm v0.2.2
│   │   └── traitlets v5.14.3
│   ├── ipython v8.12.3
│   │   ├── appnope v0.1.4
│   │   ├── backcall v0.2.0
│   │   ├── decorator v5.1.1
│   │   ├── jedi v0.19.1
│   │   │   └── parso v0.8.4
│   │   ├── matplotlib-inline v0.1.7
│   │   │   └── traitlets v5.14.3
│   │   ├── pexpect v4.9.0
│   │   │   └── ptyprocess v0.7.0
│   │   ├── pickleshare v0.7.5
│   │   ├── prompt-toolkit v3.0.47
│   │   │   └── wcwidth v0.2.13
│   │   ├── pygments v2.18.0
│   │   ├── stack-data v0.6.3
│   │   │   ├── asttokens v2.4.1
│   │   │   │   └── six v1.16.0
│   │   │   ├── executing v2.0.1
│   │   │   └── pure-eval v0.2.3
│   │   └── traitlets v5.14.3
│   ├── jupyterlab-widgets v3.0.13
│   ├── traitlets v5.14.3
│   └── widgetsnbextension v4.0.13
└── psygnal v0.11.1
```

NB: we have [a PR](https://github.com/manzt/anywidget/pull/579) experimenting with dropping ipywidgets all together for a smaller module graph.

Reply:
@manzt very cool!!

Reply:
@marthacryan coming back to my original point, we don't specify `ipywidgets` here, so I don't think we need to add `anywidget` (I _do_ think that our system for managing dev/required dependencies is quite odd, but that's out of scope of this PR and for now I'd say just leave this `reqs` unchanged).